src/windows/editor_object_selection.c

```diff
@@ -2116,19 +2169,22 @@ static bool filter_chunks(rct_object_entry *entry, rct_object_filters *filter)
 static void filter_update_counts()
 {
 	if (!_FILTER_ALL || strlen(_filter_string) > 0) {
+		int numObjects = gInstalledObjectsCount;
```

Comment:
wouldn't this make it faster for the for, as then when it checks the "i < numObjects" it will only have to check the cache/local value rather than looking for "gInstalledObjectsCount" each time?